x-pack/plugins/ml/server/routes/anomaly_detectors.ts

```diff
@@ -301,6 +301,45 @@ export function jobRoutes({ router, routeGuard }: RouteInitialization) {
     })
   );
 
+  /**
+   * @apiGroup AnomalyDetectors
+   *
+   * @api {post} /api/ml/anomaly_detectors/:jobId/_reset Reset specified job
+   * @apiName ResetAnomalyDetectorsJob
+   * @apiDescription Resets an anomaly detection job.
+   *
+   * @apiSchema (params) jobIdSchema
+   */
+  router.post(
+    {
+      path: '/api/ml/anomaly_detectors/{jobId}/_reset',
+      validate: {
+        params: jobIdSchema,
+        query: schema.object({ wait_for_completion: schema.maybe(schema.boolean()) }),
```

Comment:
worth defining a schema for the query as well to include it in the API docs

Reply:
Updated in https://github.com/elastic/kibana/pull/108039/commits/06f04dc840fb5a6cf2d98bd62bf8fa7a77f9b747